test/seekrets.bats

```diff
@@ -60,6 +60,12 @@ load test_helper
     [ $status -gt 0 ]
 }
 
+@test "git-seekrets does not find newrelic keys as aws keys in test repo" {
```

Comment:
How does this test that it didn't find a newrelic key as AWS key exactly? You add a file with a new relic secret, and then, you get a result of 1 secret found. I see nothing related to AWS in this test.